src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Commands/DeleteCountryEnumerationsTranslationsCommand.g.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Generated
+
+#nullable enable
+
+using MediatR;
+using FluentValidation;
+using Microsoft.EntityFrameworkCore;
+using Nox.Application.Commands;
+using Nox.Solution;
+using Nox.Types;
+using ClientApi.Infrastructure.Persistence;
+using ClientApi.Domain;
+using CountryEntity = ClientApi.Domain.Country;
+
+namespace ClientApi.Application.Commands;
+public partial record  DeleteCountriesContinentsTranslationsCommand(Nox.Types.CultureCode CultureCode) : IRequest<bool>;
+
+internal partial class DeleteCountriesContinentsTranslationsCommandHandler : DeleteCountriesContinentsTranslationsCommandHandlerBase
+{
+	public DeleteCountriesContinentsTranslationsCommandHandler(
+        AppDbContext dbContext,
+		NoxSolution noxSolution) : base(dbContext, noxSolution)
+	{
+	}
+}
+internal abstract class DeleteCountriesContinentsTranslationsCommandHandlerBase : CommandCollectionBase<DeleteCountriesContinentsTranslationsCommand, CountryContinentLocalized>, IRequestHandler<DeleteCountriesContinentsTranslationsCommand, bool>
+{
+	public AppDbContext DbContext { get; }
+
+	public DeleteCountriesContinentsTranslationsCommandHandlerBase(
+        AppDbContext dbContext,
+		NoxSolution noxSolution) : base(noxSolution)
+	{
+		DbContext = dbContext;
+	}
+
+	public virtual async Task<bool> Handle(DeleteCountriesContinentsTranslationsCommand command, CancellationToken cancellationToken)
+	{
+		cancellationToken.ThrowIfCancellationRequested();
+		await OnExecutingAsync(command);
+
+		var localizedEnums = await DbContext.CountriesContinentsLocalized.Where(x => x.CultureCode == command.CultureCode).ToListAsync(cancellationToken);
+		
+		if(localizedEnums == null || localizedEnums.Count == 0)
```

Comment:
```suggestion
		if(!localizedEnums.Any())
```

Reply:
This would be resolved with the PR
https://github.com/NoxOrg/Nox.Generator/pull/1216